docs/data/sql_functions.yml

```diff
@@ -665,6 +665,9 @@ json:
       // STRING: "false"
       JSON_VALUE('{"a": true}', 'strict $.b'
           DEFAULT FALSE ON ERROR)
+
+      // DOUBLE: 0.998
```

Comment:
Could you maybe turn all of the `//` for the examples of this function into `--`? The double slashes are actually incorrect for SQL, so we should fix this in a separate commit.